native/explorer/Cargo.toml

```diff
@@ -73,3 +73,6 @@ version = "0.29"
 
 [dependencies.polars-algo]
 version = "0.29"
+
+[patch.crates-io]
+arrow2 = { git = "https://github.com/jorgecarleitao/arrow2.git" }
```

Comment:
I think we may want to remove this patch, since a new version was launched. But I'm not sure if the fix we want is there.

Reply:
I checked and it has not been released yet. :(